### PR TITLE
[breakpad] update version to v2023.06.01

### PR DESCRIPTION
**Pull request by RealChuan**
- [x] Changes comply with the [maintainer guide](https://github.com/microsoft/vcpkg-docs/blob/main/vcpkg/contributing/maintainer-guide.md).
- [x] SHA512s are updated for each updated download.
- [ ] <s>The "supports" clause reflects platforms that may be fixed by this new version.</s>
- [ ] <s>Any fixed [CI baseline](https://github.com/microsoft/vcpkg/blob/master/scripts/ci.baseline.txt) entries are removed from that file.</s>
- [x] Any patches that are no longer applied are deleted from the port's directory.
- [x] The version database is fixed by rerunning `./vcpkg x-add-version --all` and committing the result.
- [x] Only one version is added to each modified port's versions file.

### CODE REVIEW

**Review by WangWeiLin-MV — APPROVED** [review]
The port usage tests pass with the following triplets:
* x64-windows

File: ports/breakpad/CMakeLists.txt [hunk 1]
```diff
@@ -24,10 +24,10 @@ if(NOT CMAKE_SYSTEM_NAME STREQUAL Android)
     file(GLOB_RECURSE LIBBREAKPAD_SOURCES src/processor/*.cc)
     if(WIN32)
         list(FILTER LIBBREAKPAD_SOURCES EXCLUDE REGEX
-            "_unittest|_selftest|synth_minidump|/tests|/testdata|/linux|/mac|/android|/solaris|microdump_stackwalk|minidump_dump|minidump_stackwalk")
+            "_unittest|_selftest|synth_minidump|/tests|/testdata|/linux|/mac|/android|/solaris|microdump_stackwalk|minidump_dump|minidump_stackwalk|disassembler_objdump.cc")
```
Comment (data-queue) [inline ports/breakpad/CMakeLists.txt:27]: Can you explain why we exclude this file?

Comment (Xia-Chuan) [reply ports/breakpad/CMakeLists.txt:27]: https://github.com/google/breakpad/commit/f6178140175800cc6385a151e7f23d6e5c4968ca

Comment (RealChuan) [reply ports/breakpad/CMakeLists.txt:27]: https://github.com/google/breakpad/commit/f6178140175800cc6385a151e7f23d6e5c4968ca

**Review by data-queue — APPROVED** [review]

**Pull request merged**